### PR TITLE
Fix broken links in contributors/devel/sig-scheduling

### PR DESCRIPTION
**Pull request by MadhavJivrajani**
- Fix broken links in `scheduler.md` and `scheduler_algorithm.md`
    - Used [this](https://github.com/kubernetes/kubernetes/blob/master/pkg/scheduler/framework/plugins/legacy_registry.go) file to refer to `predicates` and `priorities` in the dev documentation for `sig-scheduling`
    - Fixed the relative path for `Resource QoS` in `scheduler_algorithm.md`

/sig scheduling

### CODE REVIEW

File: contributors/devel/sig-scheduling/scheduler.md [hunk 1]
```diff
@@ -64,14 +64,14 @@ The Scheduler tries to find a node for each Pod, one at a time.
...
 - Second, it applies a set of "priority functions"
 that rank the nodes that weren't filtered out by the predicate check. For example, it tries to spread Pods across nodes and zones while at the same time favoring the least (theoretically) loaded nodes (where "load" - in theory - is measured as the sum of the resource requests of the containers running on the node, divided by the node's capacity).
-- Finally, the node with the highest priority is chosen (or, if there are multiple such nodes, then one of them is chosen at random). The code for this main scheduling loop is in the function `Schedule()` in [pkg/scheduler/core/generic_scheduler.go](http://releases.k8s.io/HEAD/pkg/scheduler/core/generic_scheduler.go)
+- Finally, the node with the highest priority is chosen (or, if there are multiple such nodes, then one of them is chosen at random). The code for this main scheduling loop is in the function `Schedule()` in [pkg/scheduler/core/generic_scheduler.go](http://releases.k8s.io/HEAD/pkg/scheduler/core/generic_scheduler.go#L97)
```
[the remark this answers is not in the document]
Comment (MadhavJivrajani) [reply contributors/devel/sig-scheduling/scheduler.md:67]: Oh, that makes sense. Thanks for the feedback! I'll make the required changes 😄

File: contributors/devel/sig-scheduling/scheduler.md [hunk 2]
```diff
@@ -71,8 +71,7 @@ that rank the nodes that weren't filtered out by the predicate check. For exampl
 Predicates are a set of policies applied one by one to filter out inappropriate nodes.
 Priorities are a set of policies applied one by one to rank nodes (that made it through the filter of the predicates).
 By default, Kubernetes provides built-in predicates and priorities policies documented in [scheduler_algorithm.md](scheduler_algorithm.md).
-The predicates and priorities code are defined in [pkg/scheduler/algorithm/predicates/predicates.go](http://releases.k8s.io/HEAD/pkg/scheduler/algorithm/predicates/predicates.go) and [pkg/scheduler/algorithm/priorities](http://releases.k8s.io/HEAD/pkg/scheduler/algorithm/priorities/) , respectively.
-
+The predicates and priorities code are defined in [pkg/scheduler/framework/plugins/legacy_registry.go]
```
Comment (Huang-Wei) [inline contributors/devel/sig-scheduling/scheduler.md:74]: Add a period in the end:)

**Review by Huang-Wei — COMMENTED** [review]
Just a nit. LGTM otherwise.

**Comment by Huang-Wei** [comment]
@MadhavJivrajani  would you mind squashing the commits?

**Comment by MadhavJivrajani** [comment]
> @MadhavJivrajani  would you mind squashing the commits?

Done! 😄

**Comment by Huang-Wei** [comment]
/lgtm
/approve

Thanks!

**Pull request merged**